hamilton/experimental/h_databackends.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2024 databackend contributors (MIT License)
```

Comment:
Licensing? We need to include this copyright (although @skrawcz is more of an expert here): https://github.com/machow/databackend/blob/main/LICENSE.)

Reply:
yeah, copy-pasted it here for the purpose the Draft PR. I'm guessing it's fine to keep the differently licensed (MIT) code in a separate file

Reply:
I think so? Not super worried but I'll delegate to @skrawcz. Probably best to add that in, I think the license is compatible.

Reply:
Is this code taken from that repository?

Reply:
The package is a single file. The original repo is from someone at Posit/RStudio who works on great_tables. They integrated it to great_tables without any special license since the two repos are MIT in that case

https://github.com/posit-dev/great-tables/blob/main/great_tables/_databackend.py

Reply:
Yep cool. so this notice makes sense. I'd just mention which functions /parts of this file this relates to if we're being pedantic :)